Xamarin.Forms.Platform.Tizen/Renderers/VisualElementRenderer.cs

```diff
@@ -948,11 +950,13 @@ void ApplyRotation(EvasMap map, ERect geometry, ref bool changed)
 		void ApplyScale(EvasMap map, ERect geometry, ref bool changed)
 		{
 			var scale = Element.Scale;
+			var scaleX = Element.ScaleX * scale;
+			var scaleY = Element.ScaleY * scale;
 
 			// apply scale factor
-			if (scale != 1.0)
+			if (scaleX != 1.0 || scaleY != 1.0)
 			{
-				map.Zoom(scale, scale,
+				map.Zoom(scaleX, scaleY,
```

Comment:
variable scale anymore unused

Reply:
scaleX and scaleY are determined by scale.